default_app/main.ts

```diff
@@ -89,11 +88,19 @@ function loadApplicationPackage (packagePath: string) {
     let appPath;
     if (fs.existsSync(packageJsonPath)) {
       let packageJson;
+      const emitWarning = process.emitWarning;
```

Comment:
What warning is this trying to suppress? Can we be more surgical about it? If not, can we write a comment here explaining why this is needed?

Reply:
importing a JSON file via the ESM loader is "experimental" so I'm hiding that warning otherwise it'll log for everyone everytime an Electron app is launched.  There's no other way to hide the warning afaics

Reply:
did i already ask why we can't just `JSON.parse(fs.readFile())`?